machine_management/control_plane_machine_management/cpmso-getting-started.adoc

```diff
@@ -6,37 +6,71 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-For AWS, {product-title} clusters that are created with version 4.12 or later use the Control Plane Machine Set Operator by default. 
+The process for getting started with the Control Plane Machine Set Operator depends on the state of the `ControlPlaneMachineSet` custom resource (CR) in your cluster. 
 
-To use this Operator on AWS clusters that are upgraded to {product-title} version 4.12 or later from an earlier version, you must xref:../../machine_management/control_plane_machine_management/cpmso-getting-started.adoc#cpmso-activating_cpmso-getting-started[activate the Operator].
+Clusters with an active generated CR:: Clusters that have a generated CR with an active state use the Operator by default. No administrator action is required. 
```

Comment:
I noticed later in the doc, the term "active state" is formatted where "active" is in backtics, "`Active` state", when referring the CR. I know this is more so a description but do you think that the formating with the backtics should be used in these statement as well since its still referencing the CR? For both the Active and Inactive state?

Reply:
I think not for these specific ones, but I will read over all to see if there are some places it seems better :slightly_smiling_face: